tlslite/tlsconnection.py

```diff
@@ -2619,6 +2671,10 @@ def _serverTLS13Handshake(self, settings, clientHello, cipherSuite,
                         continue
                     match = [match]
 
+                # check if the ticket version matches
+                # but with PSK we don't have a ticket, still have a binder
```

Comment:
this comment is hard to read and understand. Can you rephrase?

Reply:
changed